bookwyrm/views/feed.py

```diff
@@ -66,6 +72,7 @@ def get(self, request, tab, filters_applied=False):
                 "path": f"/{tab['key']}",
                 "annual_summary_year": get_annual_summary_year(),
                 "has_tour": True,
+                "has_read_throughs": len(readthroughs) > 0,
```

Comment:
Since we are filtering on the date as well, can we name this property `"has_summary_read_throughs"` instead?

Reply:
That does actually make sense and will avoid potential confusion in future.

I've also just realised the value could simply be `len(readthroughs)` because `0` is falsy.

@rsk2 do you mind making those two small changes? Then we'll stop bike-shedding this and I'll pull it in.

Reply:
OK, done!